solr/core/src/test-files/solr/question-answer-repository/repository.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "name": "question-answer",
+    "description": "A natural language question answering plugin",
+    "versions": [
+      {
+        "version": "1.0.0",
+        "date": "2019-01-01",
+        "artifacts": [
+          {
+            "url": "question-answer-request-handler-1.0.jar",
+            "sig": "C9UWKkucmY3UNzqn0VLneVMe9kCbJjw7Urc76vGenoRwp32xvNn5ZIGZ7G34xZP7cVjqn/ltDlLWBZ/C3eAtuw=="
+          }
+        ],
+        "manifest": {
+          "min-solr-version": "8.0",
+          "max-solr-version": "9.99",
+          "plugins": [
+            {
+              "name": "request-handler",
+              "setup-command": {
+                "path": "/api/collections/${collection}/config",
```

Comment:
Will you allow a package to run any HTTP command against the cluster? What privilege does the pkg manager run under? Imagine someone posts a malicious package "superduper-solr-package" somewhere that attempts to run a setup-command  `/api/authentication/disable` :-) or `/api/c/.system/update/foo` or some other arbitrary command?

That's one reason I think we should turn this whole deploy upside down and let Solr plugin points fetch info from the plugin instead of the plugin author hardcoding some literal api commands against the cluster. ´

Reply:
Package manager (CLI) needs write access to ZK for "add-repo", since it needs to publish the trusted public key. A user must only "add-repo" for a trusted publisher. Only then should the user expect non-malicious packages.
Once a package is already in the system, it doesn't matter if the deploy step (which is a CLI action) executes a HTTP command or does something malicious from inside the jar file.